### DISCUSSION TITLE
How to develop DApp with trust wallet andorid without ssl?

### DISCUSSION THREAD

**Discussion by zation** [trustwallet/wallet-core#3516] [Q&A]
When I use trust wallet andoird to visit my dapp development env, it requests a real ssl. It's very hard to make a local development within a real ssl. Is there anyway to get around this problem?
Thanks!

**Comment by satoshiotomakan** [comment]
Hi @vcoolish, could you take a look at the issue? It seems to be related to the Android app

**Comment by vcoolish** [comment]
> When I use trust wallet andoird to visit my dapp development env, it requests a real ssl. It's very hard to make a local development within a real ssl. Is there anyway to get around this problem? Thanks!

you can use ngrok to enable secure connection

**Reply by vcoolish** [reply]
android security preferences do not allow unsecure connection
